### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by isdanni**
**Working in Progress**

1. 只改了文字版， 而且一些链接和pics需要更新；
2. 最后国际版的网址；

所以这个pr可能得丢一段时间，好闲的时候随手开的( 🐒

### CODE REVIEW

**Comment by ksyx** [comment]
Converted to draft. Thx.

File: README.md [hunk 1]
```diff
@@ -12,111 +12,107 @@
...
 * * *
 
-## 如何参与完善 OI Wiki
+## How to contribute to OI Wiki
 
-我们非常欢迎你为 **OI Wiki** 编写内容，将自己的所学所得与大家分享。
+We always welcome you to write content for **OI Wiki** and share what you have learned with you.
 
-具体的贡献方式在 [如何参与](https://oi-wiki.org/intro/htc/)。
+The specific way to contribute is in [How to Participate](https://oi-wiki.org/intro/htc/) page.
```
Comment (Xeonacid) [inline README.md:102]:
```suggestion
The specific way to contribute is in [How to Contribute](https://oi-wiki.org/intro/htc/) page.
```
htc是它的缩写

File: README.md [hunk 2]
```diff
@@ -12,111 +12,107 @@
...
-<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="知识共享许可协议" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png" /></a><br />
-除特别注明外，项目中除了代码部分均采用<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/deed.zh"> (Creative Commons BY-SA 4.0) 知识共享署名 - 相同方式共享 4.0 国际许可协议</a> 及附加的 [The Star And Thank Author License](https://github.com/zTrix/sata-license) 进行许可。
+And you need to star the GitHub repository.
 
-换言之，使用过程中您可以自由地共享、演绎，但是必须署名、以相同方式共享、分享时没有附加限制，
-
-而且需要为 GitHub 仓库点赞（Star）。
-
-而如果你想要引用这个 GitHub 仓库，可以使用如下的 bibtex：
+And if you want to reference this GitHub repository, you can use the following bibtex:
```
Comment (ksyx) [inline README.md:115]: 下面那一行我加不到
bibtex 要改一下

File: README.md [hunk 3]
```diff
@@ -12,111 +12,107 @@
...
 
-## 开源软件供应链点亮计划 —— 暑期 2020
+## Copyright statement
 
-OI Wiki 社区正在参加由中科院软件所与华为 openEuler 项目共同举办的 [“开源软件供应链点亮计划——暑期2020”](https://isrc.iscas.ac.cn/summer2020/) 项目。 OI Wiki 也准备好了若干需求等待大家积极参与，详情请见 [「“开源软件供应链点亮计划——暑期2020”项目提案」](https://github.com/OI-wiki/OI-wiki/wiki/Open-Source-Promotion-Plan---Summer-2020#) 。如有意愿，请加入 Telegram 群组 [Project OI Wiki Summer2020](https://t.me/joinchat/GaEGzht3n19XgNcd3giLSA) 。
+<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License Agreement" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png" /></a><br />
+Unless otherwise specified, except for the code part of the project, we all use <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/deed.zh"> (Creative Commons BY-SA 4.0) Creative Commons Attribution-Share in the Same Way 4.0 International License Agreement</a> Add the additional [The Star And Thank Author License](https://github.com/zTrix/sata-license) for copyright approval.
 
-## 版权声明
+In other words, you can share freely during use, but you must sign, and share in the same way without additional restrictions.
```
Comment (ksyx) [inline README.md:111]: sign 🌚 

这个建议参考 CC

Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.

File: README.md [hunk 4]
```diff
@@ -12,111 +12,107 @@
...
+# 2. Get a static page under the site folder
 mkdocs build -v
 
-# 获取 mkdocs 的命令行工具的说明（解释了命令和参数的含义）
+# To obtain the description of the mkdocs command line tool (explains the meaning of commands and parameters), please use
 mkdocs --help
 ```
 
-我们现在在服务器端渲染 MathJax ，如果希望实现类似效果，可以参考 [.travis.yml](https://github.com/OI-wiki/OI-wiki/blob/master/.travis.yml)。（需要安装 Node.js）
+We are now rendering MathJax on the server side. If you want to achieve a similar effect, you may refer to [.travis.yml](https://github.com/OI-wiki/OI-wiki/blob/master/.travis.yml) for more details. (Need to install [Node.js](https://nodejs.org/en/download/))
```
Comment (ksyx) [inline README.md:65]: Node.js required

File: README.md [hunk 5]
```diff
@@ -12,111 +12,107 @@
...
 * * *
 
-## 部署
+## Deploy
 
-本项目目前采用 [MkDocs](https://github.com/mkdocs/mkdocs) 部署在 [oi-wiki.org](https://oi-wiki.org)。
+This project is currently deployed at [oi-wiki.org](https://oi-wiki.org) using [MkDocs](https://github.com/mkdocs/mkdocs).
 
-我们在 [status.oi-wiki.org](https://status.oi-wiki.org) 维护了一份镜像站列表，它们的内容和 [oi-wiki.org](https://oi-wiki.org) 都是相同的。
+We also maintain a list of mirror sites at [status.oi-wiki.org](https://status.oi-wiki.org). The contents and [oi-wiki.org](https://oi- wiki.org) are the same.
```
Comment (ksyx) [inline README.md:35]:
```suggestion
We also maintain a list of mirror sites at [status.oi-wiki.org](https://status.oi-wiki.org). The contents and [oi-wiki.org](https://oi-wiki.org) are the same.
```

File: README.md [hunk 6]
```diff
@@ -12,111 +12,107 @@
...
+In order to make it easier for friends who love programming competitions to get started, in July 2018, **OI Wiki** was migrated to GitHub. As the content of **OI Wiki** continues to improve, more and more friends are starting to join in.
 
-**OI Wiki** 致力于成为一个免费开放且持续更新的知识整合站点，大家可以在这里获取关于 **编程竞赛 (competitive programming)** 有趣又实用的知识，我们为大家准备了竞赛中的基础知识、常见题型、解题思路以及常用工具等内容，帮助大家更快速深入地学习编程竞赛。
+**OI Wiki** is committed to becoming a free, open and continuously updated knowledge integration website. You can find interesting and practical knowledge about **competitive programming** here. We have prepared the content including basic knowledge, common types of questions, problem-solving ideas, and common tools, etc., to help everyone learn programming competitions more quickly and deeply.
 
-目前，**OI Wiki** 的内容还有很多不完善的地方，知识点覆盖不够全面，存在一些低质量页面需要修改。**OI Wiki** 团队以及参与贡献的小伙伴们正在积极完善这些内容。
+Right now, the content of **OI Wiki** still has many imperfections, the coverage of knowledge points is not comprehensive enough, and there are some pages whose quality needs to be modified. The **OI Wiki** team and the contributing friends are actively improving these content.
```
Comment (ksyx) [inline README.md:23]: > whose quality needs to be modified.

File: README.md [hunk 7]
```diff
@@ -12,111 +12,107 @@
...
 
-为了方便热爱编程竞赛的小伙伴更好地入门，2018 年 7 月份，**OI Wiki** 迁移至 GitHub。随着 **OI Wiki** 的内容不断完善，越来越多的小伙伴参与其中。
+In order to make it easier for friends who love programming competitions to get started, in July 2018, **OI Wiki** was migrated to GitHub. As the content of **OI Wiki** continues to improve, more and more friends are starting to join in.
 
-**OI Wiki** 致力于成为一个免费开放且持续更新的知识整合站点，大家可以在这里获取关于 **编程竞赛 (competitive programming)** 有趣又实用的知识，我们为大家准备了竞赛中的基础知识、常见题型、解题思路以及常用工具等内容，帮助大家更快速深入地学习编程竞赛。
+**OI Wiki** is committed to becoming a free, open and continuously updated knowledge integration website. You can find interesting and practical knowledge about **competitive programming** here. We have prepared the content including basic knowledge, common types of questions, problem-solving ideas, and common tools, etc., to help everyone learn programming competitions more quickly and deeply.
```
Comment (ksyx) [inline README.md:21]: aimed to be

File: README.md [hunk 8]
```diff
@@ -12,111 +12,107 @@
 
 * * *
 
-## 内容
+## Content
 
-编程竞赛发展多年，难度越来越高，内容越来越复杂，而网上资料大多零散，初学者往往并不知道如何系统地学习相关知识，需要花费大量时间摸索。
+Programming competitions have been around for many years. Its difficulty level is getting higher and higher, and the content is getting more and more complicated. However, most of the online materials are scattered, which takes a lot of time to explore, and beginners often do not know how to learn relevant knowledge systematically.
```
Comment (ksyx) [inline README.md:17]: its difficulty is increasingly higher 后面那句同

File: README.md [hunk 9]
```diff
@@ -12,111 +12,107 @@
...
+The specific way to contribute is in [How to Contribute](https://oi-wiki.org/intro/htc/) page.
 
 * * *
 
-## 开源软件供应链点亮计划 —— 暑期 2020
+## Copyright statement
 
-OI Wiki 社区正在参加由中科院软件所与华为 openEuler 项目共同举办的 [“开源软件供应链点亮计划——暑期2020”](https://isrc.iscas.ac.cn/summer2020/) 项目。 OI Wiki 也准备好了若干需求等待大家积极参与，详情请见 [「“开源软件供应链点亮计划——暑期2020”项目提案」](https://github.com/OI-wiki/OI-wiki/wiki/Open-Source-Promotion-Plan---Summer-2020#) 。如有意愿，请加入 Telegram 群组 [Project OI Wiki Summer2020](https://t.me/joinchat/GaEGzht3n19XgNcd3giLSA) 。
+<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License Agreement" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png" /></a><br />
+Unless otherwise specified, except for the code part of the project, we all use <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/deed.zh"> (Creative Commons BY-SA 4.0) Creative Commons Attribution-Share in the Same Way 4.0 International License Agreement</a> Add the additional [The Star And Thank Author License](https://github.com/zTrix/sata-license) for copyright approval.
```
Comment (ksyx) [inline README.md:109]: 这一块应该有官方描述的但是那个页面我还开不出来 到时候再改

File: README.md [hunk 10]
```diff
@@ -12,111 +12,107 @@
...
-我们非常欢迎你为 **OI Wiki** 编写内容，将自己的所学所得与大家分享。
+We always welcome you to write content for **OI Wiki** and share what you have learned with you.
 
-具体的贡献方式在 [如何参与](https://oi-wiki.org/intro/htc/)。
+The specific way to contribute is in [How to Contribute](https://oi-wiki.org/intro/htc/) page.
 
 * * *
 
-## 开源软件供应链点亮计划 —— 暑期 2020
+## Copyright statement
```
Comment (ksyx) [inline README.md:106]: Licenses

**Comment by ksyx** [comment]
有多少算多少吧，先换个 readme 好看点（

File: README.md (hunk 3 above)
Comment (isdanni) [reply README.md:111]: 重新糊了一个不知道行不行 🌚 

> In other words, you may share in a reasonable manner during the use but you must sign, share in the same way without additional restrictions. And in no way should it suggest the licensor endorse you or your use.

File: README.md [hunk 11]
```diff
@@ -12,111 +12,107 @@
...
 
-目前，**OI Wiki** 的内容还有很多不完善的地方，知识点覆盖不够全面，存在一些低质量页面需要修改。**OI Wiki** 团队以及参与贡献的小伙伴们正在积极完善这些内容。
+Right now, the content of **OI Wiki** still has many imperfections, the coverage of knowledge points is not comprehensive enough, and there are some pages whose quality needs to be modified. The **OI Wiki** team and the contributing friends are actively improving these content.
 
-关于上述待完善内容，请参见 **OI Wiki** 的 [Issues](https://github.com/OI-wiki/OI-wiki/issues) 以及 [迭代计划](https://github.com/OI-wiki/OI-wiki/labels/Iteration%20Plan%20%2F%20%E8%BF%AD%E4%BB%A3%E8%AE%A1%E5%88%92)。
+For the content to be improved above, please refer to [Issues](https://github.com/OI-wiki/OI-wiki/issues) and [Iteration Plan](https://github.com) of **OI Wiki** /OI-wiki/OI-wiki/labels/Iteration%20Plan%20%2F%20%E8%BF%AD%E4%BB%A3%E8%AE%A1%E5%88%92).
```
Comment (ksyx) [inline README.md:25]:
```suggestion
For the content to be improved, please refer to [Issues](https://github.com/OI-wiki/OI-wiki/issues) and [Iteration Plan](https://github.com/OI-wiki/OI-wiki/labels/Iteration%20Plan%20%2F%20%E8%BF%AD%E4%BB%A3%E8%AE%A1%E5%88%92) of **OI Wiki**.
```

**Review by ksyx — APPROVED** [review]

**Pull request merged**